go-controller/pkg/kubevirt/types.go

```diff
@@ -2,4 +2,11 @@ package kubevirt
 
 const (
 	AllowPodBridgeNetworkLiveMigrationAnnotation = "kubevirt.io/allow-pod-bridge-network-live-migration"
+	SubnetSwitchNameLabel                        = "kubevirt.io/subnet-switch-name"
+	VMLabel                                      = "kubevirt.io/vm"
 )
+
+type IPConfig struct {
+	PoolName   string
+	Annotation string
```

Comment:
`Annotation` is too vague IMO. You can be a lot more specific naming this attribute. At least `ipAnnotation`.